### PR TITLE
New display() function that uses repr

### PR DESCRIPTION
**Pull request by takluyver**
See #6

### CODE REVIEW

**Comment by flying-sheep** [comment]
could you please add this?

```
#' @importFrom repr mime2repr
#' @importFrom repr repr_text
```

either at the top of the file like

```
#' @importFrom repr mime2repr
#' @importFrom repr repr_text
NULL

base_display <- ...
```

or to the doc comments of `display`.

---

generally i never touch NAMESPACE and always let roxygen2 do the job.

File: R/display.R [hunk 1]
```diff
@@ -72,3 +72,21 @@ display_svg = function(data = NULL, file = NULL, width = NULL, height = NULL) {
     if (!is.null(height)) metadata$height <- height
     display1('image/svg+xml', data, metadata)
 }
+
+#'Display an object using any available reprs
+#'
+#' @param obj The object to be displayed
+#' @importFrom repr mime2repr repr_text
```
Comment (takluyver) [inline R/display.R:79]: @flying-sheep you mean like this? Does it strictly need to be on two lines? I think I've seen it like this elsewhere, but maybe not. I did use roxygen.

Comment (flying-sheep) [inline R/display.R:79]: like this, and one line is indeed better :relaxed:

**Comment by takluyver** [comment]
Excellent. This one needs a manual merge now - anything else you want to review, or shall I go ahead and merge it?

**Comment by flying-sheep** [comment]
Merged, but git didn’t detect it

**Comment by takluyver** [comment]
Thanks :-)

**Pull request closed without merging**